invenio_rdm_migrator/streams/streams.py

```diff
@@ -20,8 +40,8 @@ class StreamDefinition:
     def __init__(self, name, extract_cls, transform_cls, load_cls):
         """Constructor."""
         self.name = name
-        self.extract_cls = extract_cls
-        self.transform_cls = transform_cls
+        self.extract_cls = extract_cls or NoExtract
```

Comment:
API wise Im wondering if this should be a concious decision (i.e. for to choose). If not, it could be the default of the function param.

Reply:
nvm, it would force a signature change 😢